### ISSUE TITLE
`passThrough` prop for <Can> component

### ISSUE THREAD

**Issue by stalniy** [stalniy/casl#105]
`<Can>` with that option always renders own children. It may be useful for cases when you want disable buttons and not hide them completely

```jsx
<Can I="read" a="Post" passThrough>
  (can) => <button disabled={!can}>My button</button>
</Can>
```

The similar functionality can be implemetned for Vue `<can>` component by using scoped slot

- [x] react
- [ ] vue

**Comment by stalniy** [comment]
Published in `@casl/react@0.8.0`

**Comment by stalniy** [comment]
Published in `@casl/vue@0.5.0`
